### ISSUE TITLE
My Top "X-Playlist" 

### ISSUE THREAD

**Issue by siggi1984** [z-huang/music#1323]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

The "My Top x" playlist is an automatic playlist that is created based on listening behavior (the number of songs can be freely selected in the settings). I've seen this on a ViMusic fork and think it's really cool.

Maybe you can have a look at it.

https://github.com/25huizengek1/ViTune

![350930187-cde16c13-995c-419b-8e4a-cca16b750857](https://github.com/user-attachments/assets/208fe31b-ed96-49a5-8c28-4abfb6643da0)

### Why do you want this feature?

You would always have an automatically generated (without the user having to do anything) playlist with your favorite songs within 24 hours, 1 week, 1 month, 6 months, the entire period. I think it's brilliant.

**Comment by z-huang** [comment]
Technically, it's almost the same as the stats screen, right?

**Comment by siggi1984** [comment]
> Technically, it's almost the same as the stats screen, right?

Yes, that may be true, but with this feature we would always have an automatic playlist of our favorite songs within 24 hour, 1 week, 1 month, 1 year, entire period, without the user having to do anything. In my opinion, this would be a great feature.

**Comment by z-huang** [comment]
How about placing this playlist on the home screen? Some people may want their playlist screen to be clean and not to contain generated playlists.

**Comment by siggi1984** [comment]
> How about placing this playlist on the home screen? Some people may want their playlist screen to be clean and not to contain generated playlists.

a great idea. I think it's great.
